### ISSUE TITLE
Calling Search results from « outside »

### ISSUE THREAD

**Issue by AnasGhrab** [Parisson/Telemeta#129]
Is it possible to call the results of an advanced search from another website ? In which format the results are given ?

**Comment by maths28** [comment]
Yes, but it is necessary to ElasticSearch to be available from "outside". Actually it's not the case. For this, the port 9200 must be opened to "outside"
